tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "target": "es2015",
+    "skipLibCheck": true,
```

Comment:
Why did you add this skipLibCheck removing this is causing errors. Is that inherent issue in the new dependency for social login